components/automate-gateway/handler/authz.go

```diff
@@ -250,14 +251,30 @@ func (a *AuthzServer) IntrospectAllProjects(
 }
 
 func logResult(log *logrus.Entry, endpointMap map[string]*gwAuthzRes.MethodsAllowed) {
-	if len(endpointMap) == 0 {
-		log.Debug("Allowed paths: NONE")
-		return
-	}
 	paths := make([]string, len(endpointMap))
 	i := 0
-	for k := range endpointMap {
-		paths[i] = k
+	for k, v := range endpointMap {
+		methods := []string{}
+		if v.Get == true {
```

Comment:
uhm... we could simplify this:
```suggestion
		if v.Get {
```
(same with the others below)